### ISSUE TITLE
Перемотка описания фильма невозможна

### ISSUE THREAD

**Issue by sambul13** [mpertsov/movian-lostfilm#5]
Описание фильма не всегда полность помещается справа от списка. При этом перемотать описание вниз и прочитать его оставшуюся часть невозможно. Вместо этого плагин перескакивает на описание следующего сериала.

**Comment by repa4ok** [comment]
@sambul13, не уверен что это реализуемо в movian'е: разве там есть отдельные шорткаты для скроллинга контента отдельно? есть какие-либо плагины для примера?

**Comment by sambul13** [comment]
Да, в OroroTV есть скроллинг описания фильма в правом окне, если описание длинное. Также в YTS плагине. В ряде плагинов он реализован в окнах 2-го уровня, т.е. кликнув на вариант всего сериала, попадаем в список серий этого варианта, и там для каждой серии в описании справа есть скроллинг. Только надо подождать несколько минут, пока инфа придёт из tmDB или подобных баз фильмов, если они подключены.
